distributed/diagnostics/progress_stream.py

```diff
@@ -47,7 +51,7 @@ async def progress_stream(address, interval):
             "setup": dumps_function(AllProgress),
             "function": dumps_function(counts),
             "interval": interval,
-            "teardown": dumps_function(Scheduler.remove_plugin),
+            "teardown": dumps_function(remove_plugin),
```

Comment:
Maybe dumb question but `Scheduler.remove_plugin` isn't a local scope function. Why does this change anything?

Reply:
I believe this is a workaround for https://github.com/cython/cython/issues/2972. An upstream fix in `cython` (https://github.com/cython/cython/pull/2969) should be included in the `cython=0.29.24` release

Reply:
Yeah exactly

Pickling of the local `remove_plugin` function should only include the function and module to `import`. The code itself won't need to be pickled, which avoids needing to pickle `Scheduler.remove_plugin`